package.json

```diff
@@ -79,7 +79,7 @@
     "stylelint": ">= 10"
   },
   "dependencies": {
-    "@babel/parser": "^7.8.3",
+    "@babel/parser": "^7.10.2",
```

Comment:
^ means 7.x. So users can use latest version(7.12.7 now) of @babel/parser.

If you are still using the old version of @babel/parser, check package-lock.json or yarn.lock of your project.